### ISSUE TITLE
`lit/no-this-assign-in-render` seems to false positive when you assign a local variable with the value of this.property

### ISSUE THREAD

**Issue by halkeye** [43081j/eslint-plugin-lit#170]
fine:
```
const _twitter: TemplateResult | symbol = this.twitter ? html`<li><a href=${`https://twitter.com/${this.twitter}`} target="_blank" rel="noreferrer noopener">Twitter</a></li>` : nothing;
```

fails:
```
let _twitter: TemplateResult | symbol = nothing;
if (this.twitter) {
  _twitter = html`<li><a href=${`https://twitter.com/${this.twitter}`} target="_blank" rel="noreferrer noopener">Twitter</a></li>`; // eslint warns
}
```

**Comment by 43081j** [comment]
sorry only just getting to this!

that definitely seems like a bug, ill see if i can reproduce it in a test 👍

**Comment by 43081j** [comment]
it passes locally in tests, can you give me information on the error you get from eslint?

maybe the console output?

**Comment by halkeye** [comment]
https://gist.github.com/halkeye/d89a183a92f082ccd76ea313a98c81c4

```
 npx eslint src/jio-socialmedia-buttons.ts

/home/halkeye/go/src/github.com/jenkins-infra/jenkins-io-components/src/jio-socialmedia-buttons.ts
  27:7  error  Members of `this` should not be assigned to in the render method. It is likely you should do this elsewhere instead (e.g. in `updated`)  lit/no-this-assign-in-render

✖ 1 problem (1 error, 0 warnings)
```
